vkcs/provider.go

```diff
@@ -70,6 +71,11 @@ func (c *config) NetworkingV2Client(region string, sdn string) (*gophercloud.Ser
 	return client, err
 }
 
+func (c *config) PublicDNSV2Client(region string) (*gophercloud.ServiceClient, error) {
+	client, _ := c.CommonServiceClientInit(newPublicDNSV2, region, "publicdns")
```

Comment:
Whet if a region does not have publicdns? It is possible for this service. How do we arrange this case?

Reply:
Updated (it sneaked from workaround for time when there was no record in catalog for service):
https://github.com/vk-cs/terraform-provider-vkcs/blob/0eb5e7947c41bb51fe453810073b0bb23655bf54/vkcs/provider.go#L74-L76